app/views/users/index.html.erb

```diff
@@ -1,3 +1,5 @@
+<% if logged_in? %>
```

Comment:
It would be much more efficient to use a controller filter to redirect those who aren't logged in. This is not very DRY as a solution.